cpp/test/Ice/executor/Client.cpp

```diff
@@ -28,7 +28,7 @@ Client::run(int argc, char** argv)
     { executor->execute(make_shared<ExecutorCall>(call), conn); };
     // The communicator must be destroyed before the executor is terminated.
     {
-        Ice::CommunicatorHolder communicator = initialize(argc, argv, initData);
+        Ice::CommunicatorHolder communicator = initialize(initData);
```

Comment:
This syntax works because the `CommunicatorHolder(CommunicatorPtr)` constructor is _not_ explicit. Making it explicit would be cleaner but would also break some code, in particular several tests.